dask_expr/tests/test_collection.py

```diff
@@ -456,3 +457,29 @@ def test_repartition_divisions(df, opt):
         if len(part):
             assert part.min() >= df2.divisions[p]
             assert part.max() < df2.divisions[p + 1]
+
+
+def test_drop_duplicates(df, pdf):
+    assert_eq(df.drop_duplicates(), pdf.drop_duplicates())
+    assert_eq(
+        df.drop_duplicates(ignore_index=True), pdf.drop_duplicates(ignore_index=True)
+    )
+    assert_eq(df.drop_duplicates(subset=["x"]), pdf.drop_duplicates(subset=["x"]))
+    assert_eq(df.x.drop_duplicates(), pdf.x.drop_duplicates())
```

Comment:
For tests like this I wonder if we should be relying on one giant parametrized test.

A lot of our tests are just `assert_eq(func(df), fund(pdf))`.  Maybe there is something we should be doing to make this easier / lighter weight?

Reply:
I'm also happy keeping these things separate, especially when there are lots of options like this to test.  Just a thought.

Reply:
Generally, I prefer tests where the parametrisation isn't too complicated. Historically, we had some tests in pandas where the whole logic was encoded into the parametrisation, which made debugging a real PITA when something broke